crypto/verifier_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crypto
+
+import (
+	"testing"
+
+	"github.com/google/trillian"
+)
+
+const (
+	// openssl ecparam -name prime256v1 -genkey -out p256-key.pem
+	privPEM = `-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIGbhE2+z8d5lHzb0gmkS78d86gm5gHUtXCpXveFbK3pcoAoGCCqGSM49
+AwEHoUQDQgAEUxX42oxJ5voiNfbjoz8UgsGqh1bD1NXK9m8VivPmQSoYUdVFgNav
+csFaQhohkiCEthY51Ga6Xa+ggn+eTZtf9Q==
+-----END EC PRIVATE KEY-----`
+)
+
+func TestSignVerify(t *testing.T) {
+	for _, tc := range []struct {
+		PEM      string
+		HashAlgo trillian.HashAlgorithm
+		SigAlgo  trillian.SignatureAlgorithm
+	}{
+		{privPEM, trillian.HashAlgorithm_SHA256, trillian.SignatureAlgorithm_ECDSA},
+	} {
+
+		km := NewPEMKeyManager()
+		if err := km.LoadPrivateKey(tc.PEM, ""); err != nil {
+			t.Error(err)
```

Comment:
More information in failure messages would be good (throughout); Go form seems to [normally](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures) be like `t.Errorf("MyFunc(arg)=%v; want %v", got, want)`, so this one would be:
```
t.Errorf("LoadPrivateKey(pem)=%v; want nil", err)
```